### PR TITLE
Add shuffle stats in web ui

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Add shuffle stats in web ui
Test plan - verified using HiveQueryRunner
Shuffled Stats
Annotate1
Annotate2

== NO RELEASE NOTE ==

### CODE REVIEW

File: presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx [hunk 1]
```diff
@@ -1420,6 +1420,22 @@ export class QueryDetail extends React.Component {
                                             {query.queryStats.rawInputDataSize}
                                         </td>
                                     </tr>
+                                   <tr>
+                                        <td className="info-title">
+                                            Shuffled Rows
```
Comment (mbasmanova) [inline presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx:1425]: I wonder if there is a way to add a "help question mark" and provides some details on this stat, e.g. explain that this is the total number of rows shuffled across all query stages.

**Review by abhiseksaikia — APPROVED** [review]
LGTM!

**Review by mbasmanova — APPROVED** [review]
Looks great! Thank you for adding tooltips.

**Comment by xiaoxmeng** [comment]
> Looks great! Thank you for adding tooltips.

Thanks for suggestion!

**Pull request merged**